xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -349,6 +353,9 @@ bool CRenderManager::Configure()
       return false;
   }
 
+  // configure the renderer to crop if necessary
+  m_pRenderer->ConfigureCropping(m_cropBottom, m_cropTop, m_cropLeft, m_cropRight);
```

Comment:
I did it this way because it is common functionality that I wanted in the base class. If I added it to Configure I would have had to pass the cropping rect to every BaseRenderer subclass and duplicate the same code into each Configure implementation.


    _____________________________
From: Rainer Hochecker <notifications@github.com>
Sent: Thursday, March 24, 2016 12:28 AM
Subject: Re: [xbmc] Enable cropping of video frames  (#9437)
To: xbmc/xbmc <xbmc@noreply.github.com>
Cc: Nic Wolfe <nic@wolfeden.ca>


    

In xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp:   > @@ -349,6 +353,9 @@ bool CRenderManager::Configure()>        return false;>    }>  > +  // configure the renderer to crop if necessary> +  m_pRenderer->ConfigureCropping(m_cropBottom, m_cropTop, m_cropLeft, m_cropRight);   

why an extra method? cropping rect can be passed with configure   

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub